virttest/qemu_devices/qdevices.py

```diff
@@ -1773,6 +1773,9 @@ def __init__(self, name, aobject, child_bus=None):
         self.set_param('name', name)
         self._daemon_process = None
 
+    def daemon_process(self):
```

Comment:
```suggestion
    @property
    def daemon_process(self):
```

Reply:
Plus add the docstring about this method.

Reply:
Thanks a lot, corrected.